libs/agno/agno/models/mistral/mistral.py

```diff
@@ -93,6 +92,12 @@ def _format_messages(messages: List[Message]) -> List[MistralMessage]:
             raise ValueError(f"Unknown role: {message.role}")
 
         mistral_messages.append(mistral_message)
+
+    # Check if the last message is an assistant message
+    if mistral_messages and hasattr(mistral_messages[-1], "role") and mistral_messages[-1].role == "assistant":
+        # Set prefix=True for the last assistant message to allow it as the last message
+        mistral_messages[-1].prefix = True
```

Comment:
I assume this is a mistral thing? If it works, then sure

Reply:
yes it is